docs/content/installation/metrics/metrics-configuration.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Metrics Configuration Reference"
+date:
+draft: false
+weight: 30
+---
+
+# PostgreSQL Operator Metrics Installer Configuration
+
+When installing the PostgreSQL Operator Metrics infrastructure you have various configuration options available, which
```

Comment:
Similar to earlier comment.I think we should ensure we call it "pgMonitor" as we are using that stack.

Also give intros to pgMonitor as needed, explain how it is a one-stop solution for monitoring all of the critical metrics to ensure you run a healthy PostgreSQL cluster.

Reply:
Just wanted to mention that there is a separate story to create an architecture section for the metrics infrastructure, which will provide a detailed overview of it's various components, e.g. pgmonitor.

However, I can definitely provide an intro here as well.

Reply:
I went ahead and added an intro to the "PostgreSQL Operator Monitoring" installation page, which is included in [this commit](https://github.com/CrunchyData/postgres-operator/pull/1757/commits/f178310e7979fffeec4fbc08d0346ee7cfcadcba).